lib/screens/pending_config.ex

```diff
@@ -0,0 +1,18 @@
+defmodule Screens.PendingConfig do
+  @moduledoc false
+  alias ScreensConfig.Screen
+
+  @type t :: %__MODULE__{
+          screens: %{
+            ScreensConfig.Config.screen_id() => Screen.t()
+          }
+        }
+
+  @enforce_keys [:screens]
+  defstruct @enforce_keys
+
+  use ScreensConfig.Struct, children: [screens: {:map, Screen}]
+
+  defp value_from_json(_, value), do: value
+  defp value_to_json(_, value), do: value
+end
```

Comment:
Just realized this should probably go over in screens-config-lib since the other [top level `Screens.Config`](https://github.com/mbta/screens-config-lib/blob/main/lib/config/config.ex) module has now been moved there as well.

Let me know if that sounds good and I'll do it.

Reply:
Ha! Took the words out of my mouth. (That was something I have queued up in my review.) Yeah, sounds good to me. Edit: oh shoot, GH won't let me send comments while I have a review pending. That's annoying!